frontend/README.md

```diff
@@ -23,15 +23,28 @@ docker-compose up
 ```
 
 ## Without Docker
-Follow the steps below 
+Follow the steps below
 
 1. ```git clone https://github.com/SigNoz/signoz.git && cd signoz/frontend```
 1. change baseURL to ```<test environment URL>``` in file ```src/constants/env.ts```
 
 1. ```yarn install```
 1. ```yarn dev```
 
-```Note: Please ping us in #contributing channel in our slack community and we will DM you with <test environment URL>``` 
+```Note: Please ping us in #contributing channel in our slack community and we will DM you with <test environment URL>```
+
+### To Avoid Adding a file to git or ignore changes
```

Comment:
Not sure, if we need this instructions.

Reply:
@jyash97 this is good to know instruct=ions when you want to avoid changs adding to a file

Reply:
I agree this is good to know instructions. But usually project doesnt include git instructions to remove files.

On other note: I have never used this till now 😅 

cc @nidhi-tandon @ankitnayan what are your thoughts?

Reply:
this is not to remove files, for example if you make changes in locale .env files then you dont need to commit them as they are locale
so what you can do ignore them, so any changes you do will not be added to commit

Reply:
`README.md` should contain info about the projects. Usually, OSS projects have 2 sets of users engaged a) OSS users, and b) OSS developers. The above info seems useful for b) type users. I guess `README.md` is more focussed on a) type users and `CONTRIBUTING.md` on b) type users.